plugins/mcreator-localization/help/fr_FR/dimension/condition_portal_make.md

```diff
@@ -0,0 +1,2 @@
+Si la condition est spécifiée, le joueur doit la passer afin de construire un portail vers cette dimension.
+La dépendance de la pile d'item est l'item utilisé pour l'activation du portail.
```

Comment:
Chane pile d'item by Itemstack. I'm not sure everyone will understand what it is.

Reply:
Fixed.